dbbackup/management/commands/dbbackup.py

```diff
@@ -29,6 +29,8 @@ class Command(BaseDbBackupCommand):
         make_option("-s", "--servername", help="Specify server name to include in backup filename"),
         make_option("-z", "--compress", help="Compress the backup files", action="store_true", default=False),
         make_option("-e", "--encrypt", help="Encrypt the backup files", action="store_true", default=False),
+        make_option("-m", "--mongo", help="Backup a mongo db instead of sql. Uses MONGO_SETTINGS", action="store_true",
+                    default=False),
```

Comment:
Why add a a parameter, does Dbbackup not detect which database it is ?
MongoDB usage should not be specific.